packages/neo-one-server-plugin-project/src/__data__/icojs/one/generated/sourceMaps.js

```diff
@@ -1,15 +1,15 @@
-/* @hash 6e6420d58bfda29c46d8be62f51891e8 */
```

Comment:
I've been doing `git checkout` on these changes when they come up. Its not a big deal just extra stuff on the PR.

Reply:
Good call, I'll do that and update.